tests/panels/test_template.py

```diff
@@ -109,6 +110,14 @@ def test_empty_context(self):
             ["{'False': False, 'None': None, 'True': True}"],
         )
 
+    def test_lazyobject(self):
+        response = self.panel.process_request(self.request)
+        t = Template("")
+        c = Context({"lazy": SimpleLazyObject(lambda: "lazy_value")})
+        t.render(c)
+        self.panel.generate_stats(self.request, response)
+        self.assertNotIn("lazy_value", self.panel.content)
+
```

Comment:
Please add a "positive" test to demonstrate that the value is shown when it has been evaluated before.

Reply:
Good point, it actually doesn't work as intended because the `template_rendered` signal is sent *before* the template is actually rendered. It looked like it was working for me because I forced evaluation before rendering the template… So the patch needs to be a bit more complex.